stdlib/public/Platform/SwiftBionic.h

```diff
@@ -0,0 +1,46 @@
+//===--- SwiftBionic.h ----------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2024 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef SWIFT_BIONIC_MODULE
+#define SWIFT_BIONIC_MODULE
+
+#include <complex.h>
+#include <ctype.h>
+#include <errno.h>
+#include <fenv.h>
+#include <inttypes.h>
+#include <limits.h>
+#include <locale.h>
+#include <malloc.h>
+#include <math.h>
+#include <setjmp.h>
+#include <signal.h>
+#include <stdatomic.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdio_ext.h>
+#include <stdlib.h>
+#include <string.h>
+#include <time.h>
+#include <threads.h>
+#include <uchar.h>
+#include <wchar.h>
+
+// C headers that are included with the compiler.
+#include <float.h>
+#include <iso646.h>
+#include <stdarg.h>
+#include <stdbool.h>
+#include <stddef.h>
+#include <tgmath.h>
```

Comment:
These are not part of Bionic though, are they usually required by the Bionic headers?

Reply:
No, I dropped them!